developer-docs-site/docs/nodes/validator-node/operator/staking-pool-operations.md

```diff
@@ -5,23 +5,118 @@ slug: "staking-pool-operations"
 
 # Staking Pool Operations
 
-This document describes how to perform staking pool operations. You can stake only when you met the minimal staking requirement. 
+This document describes how to perform staking pool operations. Note that you can stake only when you met the minimal staking requirement. 
 
 :::tip Minimum staking requirement
 The current required minimum for staking is 1M APT tokens.
 :::
 
-## Seeing your stake pool information
+## Initializing the stake pool
 
-To see the details of your stake pool, run the below CLI command with the `get-stake-pool` option by providing the `--owner-address` and `--url` fields. 
+Make sure that this initializing the stake pool step was performed by the owner. See [Initialize staking pool](/nodes/validator-node/owner/index#initialize-staking-pool) in the owner documentation section.
 
-:::tip Use CLI 0.3.8 or higher
-Make sure you use the CLI version 0.3.8 or higher. See [Installing Aptos CLI](/cli-tools/aptos-cli-tool/install-aptos-cli.md).
-- Type `aptos --help` to see the CLI version.
-- Type `aptos node get-stake-pool --help` for more on the command option for the below example.
+## Joining validator set
+
+:::danger Errors? 
+If you run into any errors, see the [Issues and Workarounds](/docs/issues-and-workarounds.md).
+:::
+
+Follow these steps to setup the validator node using the operator account and join the validator set.
+
+:::tip Mainnet vs Testnet
+The below CLI command examples use mainnet. See the `--rest-url` value for testnet or devnet in [Aptos Blockchain Deployments](/docs/nodes/aptos-deployments.md).
 :::
 
-The below command is an example for Premainnet and an example owner address `e7be097a90c18f6bdd53efe0e74bf34393cac2f0ae941523ea196a47b6859edb`. For other networks, use the appropriate REST URL for the `--url` field. See [Aptos Blockchain Deployments](/nodes/aptos-deployments) for `--url` field values. 
+### 1. Initialize Aptos CLI
+
+  ```bash
+  aptos init --profile mainnet-operator \
+  --private-key <operator_account_private_key> \
+  --rest-url https://fullnode.mainnet.aptoslabs.com/v1 \
```

Comment:
This is weird how some say mainnet others say testnet

Reply:
Fixed.